CONTRIBUTING.md

```diff
@@ -157,6 +158,20 @@ To pick up an issue:
 
 For continued adventures, search for issues with the label [E-help-wanted](https://github.com/filecoin-project/go-filecoin/issues?q=is%3Aopen+is%3Aissue+label%3AE-help-wanted). These are slightly thornier problems that are also reasonably well-prepared.
 
+### Pipelines (ZenHub)
+We use ZenHub pipelines to track the flow of work on open issues. 
```

Comment:
We should add a link to Zenhub: https://app.zenhub.com/workspaces/filecoin-5ab0036a12f8e82ae4ed60f0/board?repos=113219518,158620468&showPipelineDescriptions=false

Reply:
I think it's worth calling out that general contributors do not need, but committers & maintainers are expected to keep pipelines info up to date.

Reply:
The ZenHub link is a few lines above here (but not including the private repo).

I added a sentence about expectations as you suggest.